src/main/java/dk/kb/image/ColorConversion.java

```diff
@@ -94,5 +107,114 @@ public static float convertRGBtoOKlab(int RGB){
         float oklabFloat = ColorTools.fromRGBA(RGBfloatArray[0],RGBfloatArray[1],RGBfloatArray[2],RGBfloatArray[3]);
         return oklabFloat;
     }
-    
+
+    /**
+     * Calculates which color from the chosen palette each color from the RGB color space is closest to.
+     * Saves the calculated entries as raw bytes.
```

Comment:
Please describe what this is used for. If not, this code looks like some leftover code from previous experiments.